packages/theming/theme-types/src/Theme.types.ts

```diff
@@ -46,7 +44,7 @@ export type PartialTheme = Omit<TwoLevelPartial<Theme>, 'typography' | 'host'> &
   host?: TwoLevelPartial<Theme['host']>;
 };
 
-export type AppearanceOptions = 'light' | 'dark';
+export type AppearanceOptions = 'light' | 'dark' | 'highContrast';
```

Comment:
Curious, does the React Native Appearance API on windows/win32 show HC as an option? If so, would it be safer to use the one coming from RN itself instead of defining it again here?

Reply:
No, it doesn't. RNW doesn't seem to use appearance, it uses AppTheme for light/dark and isHighContrast separately.

Reply:
(I don't really know what win32 does, I'll look into it)